crates/build/src/manifest.rs

```diff
@@ -29,3 +46,42 @@ pub(crate) struct RawBuildConfig {
     pub command: String,
     pub workdir: Option<PathBuf>,
 }
+
+/// FixedVersion represents a schema version field with a const value.
+#[derive(Clone, Debug, Default, Serialize, Deserialize)]
+#[serde(into = "usize", try_from = "String")]
+pub struct FixedVersion<const V: usize>;
```

Comment:
`spin-build` already depends on `spin-loader`, so it should probably import `FixedStringVersion` from there.